charts/templates/crds/clusters.clusterpedia.io_pediaclusters.yaml

```diff
@@ -0,0 +1,222 @@
+
```

Comment:
For the management of crds in charts, can we look at cert-manager or is there a better solution?
ref: 
* https://cert-manager.io/docs/installation/helm/#option-2-install-crds-as-part-of-the-helm-release
* https://cert-manager.io/docs/installation/upgrading/#upgrading-with-helm

Reply:
I looked at the cert-manager, the flag `installCRDs` is used to determine whether or not to install CRD in updation.

This a useful flag to us.